src/amo/utils/index.js

```diff
@@ -42,25 +42,32 @@ export function getAddonURL(slug: string): string {
   return `/addon/${slug}/`;
 }
 
+// Note that this also excepts an object with any number of query params which
+// are also added to the final querystring.
+// Note also that anything in otherQueryParams will overried any specific utm
+// params.
 export const makeQueryStringWithUTM = ({
   utm_campaign = DOWNLOAD_FIREFOX_UTM_CAMPAIGN,
   utm_content = null,
   utm_medium = DEFAULT_UTM_MEDIUM,
   utm_source = DEFAULT_UTM_SOURCE,
   utm_term = null,
+  otherQueryParams = {},
```

Comment:
My initial thought was to collect all the other query params into this variable (using the rest syntax) but I guess that works too.

Reply:
I tried that, but Flow was getting confused. I think this works just as well.

Reply:
Edit: I also think it wasn't working as well.